modules/schedulers/src/main/resources/META-INF/mule-schedulers.xsd

```diff
@@ -26,6 +26,13 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="timeZone" type="xsd:string">
```

Comment:
It already does defined as it is. The cron expression is declared in the same way and it is tested to support expressions.